internal/host/hostcommands/install_cmd.go

```diff
@@ -181,6 +182,22 @@ func (i *installCmd) getFullInstallerCommand(cluster *common.Cluster, host *mode
 		shellescape.QuoteCommand(installerCmd)), nil
 }
 
+func (i *installCmd) getMustGatherArgument(mustGatherMap versions.MustGatherVersion) string {
+	//for backward compatability, if must gather images map contains only the ocp must gather
```

Comment:
Probably worth adding a test for this logic.

Reply:
Added